### ISSUE TITLE
'High Contrast White' theme Asset Upload WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1032]
**Describe the bug**
The 'Asset Upload' component color contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.10.0-rc.1

**To Reproduce**
1. Go to https://deploy-preview-1025--fundamental-styles.netlify.app/components/asset-upload.html
2. Check the 'Upload Link' text and icon color contrast with the background

**Expected behavior**
If we want WCAG AAA compliance, the 'Upload Link' text and icon color contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81855923-ddb91400-9514-11ea-8f94-0ef71cc914d8.png)
![image](https://user-images.githubusercontent.com/43764832/81857244-d430ab80-9516-11ea-94a4-145278aa4e26.png)
![image](https://user-images.githubusercontent.com/43764832/81739389-222eac00-9450-11ea-9224-2cf9d75fee64.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Asset Upload is a deprecated non-Fiori component that has been replaced by File Uploader.
https://github.com/SAP/fundamental-styles/pull/1187
